Library/Homebrew/os/linux/ld.rb

```diff
@@ -0,0 +1,75 @@
+# typed: true
+# frozen_string_literal: true
+
+module OS
+  module Linux
+    # Helper functions for querying `ld` information.
+    #
+    # @api private
+    module Ld
+      module_function
```

Comment:
I recommend removing this, making the methods class defs (`def self.`), and removing the `.rbi` file.